frontend/src/app/shared/components/dashboard/dashboard.component.html

```diff
@@ -1,9 +1,9 @@
-<bf-navigation></bf-navigation>
+<bf-navigation (toggleTweetInput)="onToggleTweetInput()"></bf-navigation>
```

Comment:
rename toggleTweetInput -> tweetInputClicked
Name of event emitter variable should say what happened